docs/reference/guard_get_installer.md

```diff
@@ -26,13 +26,20 @@ guard get installer [flags]
 ```
       --addr string                          Address (host:port) of guard server. (default "10.96.10.96:443")
       --auth-providers strings               name of providers for which guard will provide authentication service (required), supported providers : Azure/Github/Gitlab/Google/Ldap/Token-Auth
+      --authz-providers strings              name of providers for which guard will provide authorization service (optional), supported providers : Azure
       --azure.aks-token-url string           url to call for AKS OBO flow
       --azure.auth-mode string               auth mode to call graph api, valid value is either aks, obo, or client-credential (default "client-credential")
       --azure.client-id string               MS Graph application client ID to use
       --azure.client-secret string           MS Graph application client secret to use
       --azure.environment string             Azure cloud environment
       --azure.tenant-id string               MS Graph application tenant id to use
       --azure.use-group-uid                  Use group UID for authentication instead of group display name (default true)
+      --azure.aks-authz-url string           url to call for AKS Authz flow
+      --azure.authz-mode string              authz mode to call RBAC api, valid value is either aks or arc
+      --azure.resource-id  string            azure cluster resource id (//subscription/<subName>/resourcegroups/<RGname>/providers/Microsoft.ContainerService/managedClusters/<clustername> for AKS or //subscription/<subName>/resourcegroups/<RGname>/providers/Microsoft.Kubernetes/connectedClusters/<clustername> for arc) to be used as scope for RBAC check
+      --azure.skip-authz-check strings       name of emails for which authz check will be skipped.
+      --azure.skip-authz-for-non-aad-users   If true, Azure RBAC will be skipped for certificate users (non AAD users). Setting to false, all non AAD user requests will be denied.
```

Comment:
```suggestion
      --azure.skip-authz-for-non-aad-users   If true, Azure RBAC will be skipped for non AAD users such as certificate users and service accounts. Setting to false, all non AAD user requests will be denied.
```

Reply:
service accounts skipping is not coming under this flag as we always need to skip them.

Reply:
oh right! probably mention that in the doc as well

Reply:
i mean to explicitly to call it out